pytest_logger/plugin.py

```diff
@@ -230,7 +243,7 @@ class Formatter(logging.Formatter):
         logging.DEBUG: 'dbg',
     }
 
-    def __init__(self, fmt):
+    def __init__(self, fmt='%(asctime)s %(levelshortname)s %(name)s: %(message)s'):
```

Comment:
Since DefaultFormatter would never get instantiated with arguments, consider putting format as class attribute.

Reply:
666aa66 integrates the suggested change.